daedalus/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "output/Daedalus.ClientApi/index.js",
   "scripts": {
     "start": "npm run build:dev",
-    "preinstall": "which psc || which purs || npm install purescript@0.11.5",
+    "preinstall": "type -P psc || type -P purs || npm install purescript@0.11.5",
```

Comment:
@sectore Do you know anything about this? Do you know somebody (maybe @DominikGuzei ) which can advise? (context, we are trying to understand if differences between `cardano-sl-1.0-exchanges` and `master` are crucial or not).